README.md

```diff
@@ -71,16 +71,17 @@ try {
     await connection.transmit(someOtherCommand);
     …
     await connection.transmit(finalCommand);
-    // The transaction with the card ends when the Promise settles.
+
+    // Reset the card when ending the transaction.
+    return "reset";
   });
-  // |result| has the value the Promise returned by callback above settled with.
 } catch (ex) {
   // Either the transaction failed to start, the callback
   // has thrown or the transaction failed to end.
 }
 ```
 
-Upon calling `startTransaction()`, a transaction will be started with the card related to this connection. When successful, the given callback will be called. All interactions with the card done inside this callback will then be part of this transaction. The transaction is ended once the `Promise` returned by the callback settles.
+Upon calling `startTransaction()`, a transaction will be started with the card related to this connection. When successful, the given callback will be called. All interactions with the card done inside this callback will then be part of this transaction. The transaction is ended once the `Promise` returned by the callback settles. If the settled `Promise` is fulfilled, the transaction is ended with the card disposition it was fulfilled with. Otherwise the transaction is ended without any further action.
```

Comment:
Does it make sense to be explicit that if an invalid return value is encountered then a value of "leave" will be assumed?

Reply:
Sure. Done.